fontscan/fontmap_cache_android.go

```diff
@@ -0,0 +1,10 @@
+//go:build android
```

Comment:
I don't think we need this comment as the file name already includes `_android` suffix.

Reply:
That is true it is not needed... just a habit of mine sorry